.taskcluster.yml

```diff
@@ -35,6 +35,16 @@ tasks:
             $if: 'tasks_for == "github-pull-request"'
             then: "refs/pull/${event.number}/merge"
             else: "${event.after}"
+          owner:
```

Comment:
(Grr cant comment on the actual line I want to because its not in the diff... GitHub!)

I still don't understand why line 46 (old diff)/56 (new diff) won't cause Taskcluster problems - `event.sender.login` forms an invalid email for GitHub actions does it not?

Reply:
Yes, that will be a problem if we make actions that create PRs. I was sort of hoping that https://github.com/taskcluster/taskcluster-rfcs/pull/153 would be implemented before we do that, but maybe I should just write something that always works.

Reply:
I'm not sure what the link is to PRs? Line 56 seems to run on pushes and PRs, and for epochs the push is done by an action, so... how does that work?

Reply:
Oh, right sorry you mean I forgot to actually *use* the `owner` variable. Fixed now :)